recipes/openff-recharge/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "openff-recharge" %}
+{% set version = "0.0.1a4" %}
```

Comment:
```suggestion
{% set version = "0.0.1a6" %}
```

Reply:
Oh, this reminds me I found how to publish alphas / betas / rcs in conda-forge. See https://github.com/conda-forge/cfep/blob/master/cfep-05.md. We might want to "break" recharge on purpose so it doesn't upload anything to `main` upon rendering.